ixofoundation/ixo-webclient
# chore(entity): header override

## Scope

https://nonaredteam.atlassian.net/browse/IXO-628
https://nonaredteam.atlassian.net/browse/IXO-680

## Work Done

- Override property for the header
- Allow filter to be single select

## Steps to Test

- See that the Stage filter is Single Select
- See that when you choose the 3rd option ("Performing") on Stage it changes the header as per the settings in the schema

## Screenshots

![image](https://user-images.githubusercontent.com/20910282/84507761-9ebbe100-acc1-11ea-9fd0-31f94e4840ca.png)
